docs/reference/modules/scripting/using.asciidoc

```diff
@@ -110,12 +110,15 @@ second version is only compiled once.
 [[modules-scripting-file-scripts]]
 === File-based Scripts
 
-To increase security, non-sandboxed languages can only be specified in script
-files stored on every node in the cluster.  File scripts must be saved in the
-`scripts` directory whose default location depends on whether you use  the
-<<zip-targz-layout,`zip`/`tar.gz`>> (`$ES_HOME/config/scripts/`),
-<<rpm-layout,RPM>>, or <<deb-layout,Debian>> package.  The default may be
-changed with the `path.script` setting.
+To increase security, scripts for languages that are not deemed to be safe by
+default can only be specified in files stored on every node in the cluster. File
```

Comment:
If we go back to using "sandboxed" as the descriptor, then I'd update this para too.